ciao-4.11/contrib/bin/blanksky_image

```diff
@@ -216,6 +224,11 @@ def bkg_img(imgfile,bkg,outfile,dmfilter,dettype,chips,tmpdir,verbose):
 
     cr.__del__()
```

Comment:
I'm going to remove this - the call to `cr.__del__()` - as it should no longer be needed (I believe it's a hold-over from a crates bug that was fixed a while ago).